### DISCUSSION TITLE
Power BI Kick-off!

### DISCUSSION THREAD

**Discussion by fboucher** [microsoft/Power-Platform-For-Beginners#6] [Power BI]
Do you have a section of the [template](https://github.com/microsoft/Power-Platform-For-Beginners/tree/main/templates) you would like to start working on?
Let your teammates know!

Then you can **Fork** the repo and start creating content!

**Comment by brianbonk** [comment]
Hi there
Besides the current great headlines for Power BI from the current templates:
- Understanding data visualization principles and best practices.
- Connecting to various data sources and transforming data.
- Creating interactive reports and dashboards.
- Implementing data modeling and calculations.
- Sharing and collaborating on Power BI content.

We can probably find a lot of existing content for the above list from the current MS learn content.

I also think we should add topics (perhaps more will come later 😊):
- Embedding Power App in Power BI
- Filtering Power App from Power BI
- Writeback to database from embedded Power App and updating Power BI report in the fly

**Reply by brianbonk** [reply]
The title is "for beginners" and my last 3 bullets could be to advanced for the beginners level?
Let me know what you think...

**Comment by fboucher** [comment]
Hey Brian!
the audience targeted is beginner, but I totally agree with you. The titles were there for inspiration... I like your proposal. let's add those 3 more "advanced" at the end.

Frank

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by fboucher** [comment]
I'll close this discussion now that most happens on Teams and keep only the more general ones open for now. :)
